common/mariadb-galera/helm/scripts/mariadb-galera/readiness.sh

```diff
@@ -5,6 +5,22 @@ set -o pipefail
 
 source /opt/${SOFTWARE_NAME}/bin/common-functions.sh
 
+function checkdblogon {
```

Comment:
why not parameterising the common function and using it here?

Reply:
We can do that refactoring in a dedicated story...